static/js/constants_test.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import { assert } from 'chai';
+
+import { DASHBOARD_RESPONSE } from './constants';
+
+describe('constants', () => {
+  it("doesn't duplicate any id numbers within the same type of information", () => {
+    let programIds : Set<number> = new Set();
+    let courseIds : Set<number> = new Set();
+    let runIds : Set<number> = new Set();
```

Comment:
We don't have flow typechecking turned on for any test files, do you think this is something we should do? I had avoided it because I thought it might be a little onerous.